src/containers/Header/Header.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import HeaderTabs from '../../components/HeaderTabs';
+import Logo from '../../components/Logo';
+import Search from '../../components/Search';
+
+import { HeaderWrapper, SearchWrapper, TabsWrapper } from './style';
+
+export class Header extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: 0,
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
I don't want to bind `this` to all functions in classes. It looks a bit ugly. So, if you want to write `class` components let's add `@babel/plugin-proposal-class-properties` plugin and then you can write methods inside class without binding. Or another suggestion: you can rewrite this component using function (and hooks)